content/home/examples/a-simple-component.md

```diff
@@ -3,6 +3,6 @@ title: Un componente simple
 order: 0
 domid: hello-example
 ---
-Los componentes de React implementan un método `render()`, el cual toma los datos de entrada y regresa lo que se tiene que mostrar. En este ejemplo usaremos sintaxis tipo XML llamado JSX. Los datos de entrada que son pasados al componente pueden ser accesados en `render()` usando `this.props`.
+Los componentes de React implementan un método `render()`, el cual toma datos de entrada y devuelve lo que se tiene que mostrar. En este ejemplo usaremos sintaxis similar a XML llamado JSX. Los datos de entrada que son pasados al componente son accesibles en `render()` usando `this.props`.
```

Comment:
I sugesst turning:
> "... sintaxis similar a ..."

to

> ... una sintaxis similar a ...

adding `una`.

Reply:
Makes sense